pkg/controllers/oauthendpoints/oauth_endpoints_controller.go

```diff
@@ -224,7 +224,8 @@ func getOAuthRouteTLSConfig(cmLister corev1listers.ConfigMapLister, secretLister
 	}
 
 	return &tls.Config{
-		RootCAs: rootCAs,
+		RootCAs:            rootCAs,
+		InsecureSkipVerify: true, // TODO: REMOVE THIS! Don't merge this to OpenShift, hack to support self signed certs.
```

Comment:
This must be removed before the PR is merged.
/hold

Reply:
+1 :slightly_smiling_face:

Reply:
Removed in the latest version of the PR.